index.js

```diff
@@ -63,7 +63,7 @@ function generateBarChart(percent, size) {
   const frac = (size * 8 * percent) / 100;
   const barsFull = Math.floor(frac / 8);
   const semi = frac % 8;
-  const barsEmpty = size - barsFull - 1;
+  const barsEmpty = Math.max(size - barsFull - 1, 0);
```

Comment:
there should be a better way than this. but I don't know why there is a `-1` and I don't want to miss anything